run_test.go

```diff
@@ -137,3 +137,18 @@ func TestRunExitCode(t *testing.T) {
 	assert.Equal(base.T, "exited", inspect123.State.Status)
 	assert.Equal(base.T, 123, inspect123.State.ExitCode)
 }
+
+func TestRunCIDFile(t *testing.T) {
```

Comment:
```
=== RUN   TestRunCIDFile
    run_test.go:145: assertion failed: expression is false: res.ExitCode == exitCode: time="2021-05-24T17:38:22Z" level=fatal msg="image name needs to be specified"
        
--- FAIL: TestRunCIDFile (0.02s)
```

Reply:
Replaced by just asserting failure if the file exists. Is that enough or should it be more specific?